### DISCUSSION TITLE
Destroy mudrator nest mission doesn't spawn eggs normaly

### DISCUSSION THREAD

**Discussion by Evolved-d** [FakeFishGames/Barotrauma#15300] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

While playing quite a few of those missions in campaign i've noticed that some mudraptor eggs in the cave doesn't spawn like a structure, instead they spawn as handheld item. As there are two types of eggs in caves: hatchling's eggs and "destructable" ones, there is a clear picture of this bug: hatchling's eggs spawn as structure and then break, spawning hatchling - behaving normally, and the "main mission" eggs (or however you would like to call them) - they _used to_ spawn as structure and be destroyed by player to clear the mission, but now they simply dont spawn this way. Instead there will be just lying there. You still can destroy them and clear the mission, but:

1. While dealing with them with plasmacutter eggs get blown by the thrust, which is simply annoying;
2. And my own problem - as a lootgoblin i firstly grab them and only then remember that those need to be destroyed :/
(_Maybe in tests i will try to deconstruct them_)

### Reproduction steps

_I have not yet tried this myself_

1. Start any game with mudraptor nest mission - campaign/missions mode
2. Get to the cave and fight your way to the nest
3. Observe the issue 

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Evolved-d** [comment]
Tested this in mission mode, issue is reproducible
![image](https://github.com/user-attachments/assets/4e317faf-d906-4ca6-acbc-7280fbb6cde0)

**Comment by A2ur3** [comment]
they are for traitor missions and deconstructing or keeping them inside submarine can complete the mission

**Reply by Evolved-d** [reply]
Didn't know that, thanks for info
